src/test/mod.rs

```diff
@@ -313,6 +313,11 @@ fn self_tests() {
     );
 }
 
+fn is_nightly() -> bool {
```

Comment:
there is a macro `is_nightly_channel` in `crate::config::config_type` already which can be reused.

Reply:
Addressed in 4745cec (in new PR)